docs/Release_notes.md

```diff
@@ -1,6 +1,42 @@
 V2X-Hub Release Notes
 ---------------------------------
 
+Version 7.9.0, released Sep 24th, 2025
+--------------------------------------------------------
+
+**Summary:**
+V2X Hub release 7.9.0 includes general system improvements to documentation, configuration, deployment process, and local development environment. In addition, this release includes experimental* enhancements to integrate V2X Hub with Multimodal Inteligent Traffic Singal Systems (MMITSS) and USDOT Operation Data Environment (ODE).
+
+Enhancement in this release:  
+
+V2X Hub documentation has been updated by adding the V2X Hub Plugin Programming Guide in README.md under src/v2i-hub. The objective is to ensure that all documentation is current, accurately reflects the latest product capabilities, and is organized for easy accessibility.
+- Pull Requests: [V2X-Hub PR #644](https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/644) 
+
+Updated ODE Plugin to support TIM and MAP messages and new UDP based ODE communication interface. [Experimental] 
+- Pull Requests: [V2X-Hub PR #656](https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/656), [V2X-Hub PR #662](https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/662)
+
+CARMA Streets/V2X Hub integration with MMITSS to support Transit Signal Priority and Emergency Vehicle use cases. [Experimental] 
+- Pull Requests: [V2X-Hub PR #688](https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/688) , [V2X-Hub PR #708](https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/708) 
+
+
+Fixes in this release: 
+
+- GitHub Issue #624: Add New license information directly to V2X Hub GitHub Repository. 
+- Pull Requests: [V2X-Hub PR #636](https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/636)
+
+Other Fixes:  
+
+- [V2X-Hub PR #659](https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/659): Fixed dev container syntax and set default mode to simulation=false to avoid by default deploying V2X Hub in simulation mode. 
+- [V2X-Hub PR #660](660): Fixed build and coverage scripts to only create build and coverage information when parameter is present. 
```

Comment:
@SaikrishnaBairamoni Did this link not work properly?

Reply:
updated